compiler/ml/typecore.ml

```diff
@@ -2401,7 +2401,7 @@ and type_expect_ ?type_clash_context ?in_function ?(recarg = Rejected) env sexp
     type_function ?in_function ~arity ~async loc sexp.pexp_attributes env
       ty_expected l
       [Ast_helper.Exp.case spat sbody]
-  | Pexp_apply {funct = sfunct; args = sargs; partial} ->
+  | Pexp_apply {funct = sfunct; args = sargs; partial; transformed_jsx} ->
```

Comment:
From here on, pass it along to the next layer.

Reply:
Looks like it never makes it here. So it must have gotten lost somewhere.
I'd start with a simple test examples and start adding some assert false whenever `transformed_jsx` is not `None`, to catch where was the last point where it was present before getting lost.

Reply:
Oh looks like it gets lost in `bs_ast_mapper`.

Reply:
Yeah, it seems to take a different route because `<div>` is an FFI call.